pwa/src/apiService/resources/endpoint.ts

```diff
@@ -0,0 +1,23 @@
+import { AxiosInstance, AxiosResponse } from "axios"
+
+export default class Endpoint {
+  private _instance: AxiosInstance
+
+  constructor (_instance: AxiosInstance) {
+    this._instance = _instance
+  }
+  public getAll = (): Promise<AxiosResponse> => {
+    return this._instance.get('/attributes')
+  }
```

Comment:
You're retrieving `/attributes` and not `/endpoints` here. Resulting in the wrong response.